src/app/nonprofit/nonprofit-page/donation-dialog.html

```diff
@@ -0,0 +1,124 @@
+                    <div class="donation-input">
+                        <mat-form-field>
+                            <mat-label>Enter Amount to Donate</mat-label>
+                            <input formControlName="donationAmount" matInput type="tel" currencyMask>
+                        </mat-form-field>
+                    </div>
+                </div>
+                <button mat-button matStepperNext>Next</button>
+            </form>
+        </mat-step>
+
+        <!-- Your Information Stepper -->
+        <mat-step [stepControl]="donorInformationForm">
+            <form [formGroup]="donorInformationForm">
+                <ng-template matStepLabel>Your Information</ng-template>
+                
+                <div class="user-information-parent">
+                    <div class="inputs-container">
+                        <div class="donation-input">
+                            <!-- Card Holder Name -->
+                            <mat-form-field>
+                                <mat-label>Card Holder Name</mat-label>
+                                <input formControlName="cardHolderName" matInput type="text" >
+                            </mat-form-field> 
+                            
+                            <!-- Expiration Date-->
+                            <mat-form-field>
+                                <mat-label>Card Expiration Date</mat-label>
+                                <input formControlName="cardExpirationDate" matInput type="tel" ccExp>
+                            </mat-form-field>
+
+                        </div>
+                    </div>
+                    <div class="inputs-container">
+                        <div class="donation-input">
+                            <!-- Card Number -->
+                            <mat-form-field>
+                                <mat-label>Card Number</mat-label>
+                                <input formControlName="cardNumber" matInput type="tel" ccNumber>
+                                <mat-hint> Hint: Do NOT input real CC number</mat-hint>
+                            </mat-form-field>
+
+                            <!-- Card CVC-->
+                            <mat-form-field>
+                                <mat-label>CVC</mat-label>
+                                <input formControlName="cardCVC" matInput type="tel" ccCVC>
+                            </mat-form-field>
+                        </div>
+                    </div>
+                </div>
+                <div>
+                    <button mat-button matStepperPrevious>Back</button>
+                    <button mat-button matStepperNext>Go To Confirmation Page</button>
+                </div>
+                
+            </form>
+        </mat-step>
+
+        <!-- Confirmation Stepper -->
+        <mat-step>
+            <form [formGroup]="confirmationForm">
+                <ng-template matStepLabel>Confirm Your Donation</ng-template>
+
+                <div>
```

Comment:
[SUGGESTION] Add an organizational reminder to the confirmation so users can easily confirm which organization their donation is going to without going to the header